### DISCUSSION TITLE
Trane XR524 should match XL624

### DISCUSSION THREAD

**Discussion by cweakland** [zwave-js/zwave-js#1724] [Q&A]
I am in the planning phases of moving over to Zwave-JS, one thing I noticed was that the configuration for the Trane XR524 looks to be very complete, where as the XL624 is quite empty. These two devices are nearly identical. In fact on the old HASS Zwave integration, I just copied the xr524 device configuration to the xl624 device configuration and it works. I own both of these devices and I am willing to help verify this.

Question, how can I modify and test this prior to have it committed in the code? Like I said I am currently planning my migration and checking that everything will work. I have not yet attempted the move to zwave-js, so any pointers as to what files on HassOS I need to change to test this would be helpful. Hopefully this weekend I can get zwave-js up and attempt to test these. 

Here are the two devices in question:

https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x008b/trane_xr524.json
https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x008b/xl624.json

Edit, thanks for all your work on this!!!

**Comment by AlCalzone** [comment]
@blhoward2
